homeassistant/components/climate/radiotherm.py

```diff
@@ -86,10 +102,12 @@ def __init__(self, device, hold_temp, away_temps):
         self._tmode = None
         self._tstate = None
         self._hold_temp = hold_temp
+        self._hold_set = False
         self._away = False
-        self._away_temps = away_temps
+        self._away_temps = [self.round_temp(i) for i in away_temps]
```

Comment:
Create a function from `round_temp` and use that inside config validate. So you don't need do that here